extension/content/enginesView.css

```diff
@@ -2,38 +2,19 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#locale-region-settings {
-  display: grid;
-  justify-content: space-between;
-  align-items: center;
-  width: max-content;
-  margin: 0 auto;
-  grid-template-rows: repeat(2, auto);
-  grid-template-columns: repeat(4, calc(25% - 6em / 4));
-  column-gap: 2ch;
-  width: 80%;
-}
-
-#application-name-wrapper,
-#region-select-wrapper,
-#locale-select-wrapper {
-  grid-row: 1 / 3;
+#locale-region-settings > table {
+  width: 100%;
+  border-spacing: 2ch 0;
```

Comment:
I noticed you're using `ch` as the unit here. I'm concerned it might introduce inconsistencies across different operating systems, browsers, or custom fonts since it's based on the width of the character '0'. It might lead to unexpected layouts. It's not a huge problem here and we might not really notice, but I think it's better practice to use something like `rem`. 

`rem` is more stable because it’s always relative to the root `<html>` font size. For most browsers, including Firefox, the default is 16px.

I tried your patch with `2rem` and it seems to work fine.

Reply:
I'm using the `ch` as we're handling width, not a height and it seemed it would be more consistent relative to the text.

However, as we're not framing text, but the space between it, we're probably safe just to go with `px`.